extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -444,6 +444,9 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 				{
 					'sessionId': this._sessionId,
 					'performanceDataSource': this._skuRecommendationPerformanceDataSource,
+					'scalingFactor': this._skuScalingFactor?.toString(),
+					'skuTargetPercentile': this._skuTargetPercentile?.toString(),
+					'skuEnablePreview': this._skuEnablePreview?.toString(),
```

Comment:
nit: maybe rename to `enablePreviewFeatures` or `enablePreviewSkus` to be more consistent with the user-facing naming?